ci-operator/step-registry/baremetal/lab/pre/provisioning-network/baremetal-lab-pre-provisioning-network-commands.sh

```diff
@@ -62,6 +63,7 @@ for switch_address in switches:
     switch_port = switch_address.split(":")[1]
     with Device(host=switch_hostname, port=switch_port, user='admin', ssh_private_key_file=ssh_key_path) as dev:
         with Config(dev, mode="private") as cu:
+            print(f"Create the vlan {vlan_name} vlan-id {vlan_id}")
```

Comment:
@zhaozhanqi do you mind adding the following at line 47 for the future?

```python

if not vlan_id.isdigit():
    print(f"The vlan_id is not an integer: {vlan_id}. Verify that the reservation step allocated VIPs for the cluster (use RESERVE_BOOTSTRAP: 'false' in your test config)")
```

Reply:
sure, I will update it later